### ISSUE TITLE
ResizeObserver loop completed with undelivered notifications.

### ISSUE THREAD

**Issue by Mariusthvdb** [home-assistant/frontend#9022]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

clicking either of the below input_numbers, to reveal the more-info, show the error in the inspector. Many others don't , but these 2 do.

### Describe the behavior you expected

no error

### Steps to reproduce the issue

1. click the entity for more-info
2.
3.
...

### What version of Home Assistant Core has the issue?

2021.4.6

### What was the last working version of Home Assistant Core?

not sure

### In which browser are you experiencing the issue with?

MAc Safari, Chrome haș no issue

### Which operating system are you using to run this browser?

Big sur 11.2.3 Safari 14.0.3 

### Problem-relevant frontend configuration

```yaml
- type: custom:slider-entity-row
        entity: input_number.ha_delayed_startup
      - input_number.reload_delay
```

### Javascript errors shown in your browser console/inspector

```txt
ResizeObserver loop completed with undelivered notifications.
```

**Comment by bramkragten** [comment]
You can safely ignore this.

**Comment by Mariusthvdb** [comment]
Ok thanks.
Any thoughts on why this is happening in only that place on the config ? As you can see nothing special is in the card.

**Comment by Mariusthvdb** [comment]
seeing this in the logs on startup for beta 2, is it related, or do you need a separate issue for this:
```
2021-04-30 07:48:11 ERROR (MainThread) [frontend.js.latest.202104071] https://mydomain.duckdns.org:port/frontend_latest/chunk.74274bf73ccf9b9f0934.js:61:17215 TypeError: Argument 1 ('target') to ResizeObserver.unobserve must be an instance of Element
```

**Comment by Vameleon** [comment]
Started seeing the same error just recently. I am also experiencing some delaying in loading time of some sensor graphs (not sure of this caused by the error).

![image](https://user-images.githubusercontent.com/35732557/122981394-ff343280-d399-11eb-8cda-cd77fe5b648a.png)

![image](https://user-images.githubusercontent.com/35732557/122981656-491d1880-d39a-11eb-835d-3c9055d69f50.png)

**Comment by rodpayne** [comment]
I also started seeing this on core-2021.6.5.  When it happens, it happens about a dozen times in a row (with the same time stamp).
`2021-07-04 07:33:27 ERROR (MainThread) [frontend.js.latest.202106030] :0:0 ResizeObserver loop completed with undelivered notifications.`

**Comment by Final-Hawk** [comment]
I have the same issue, everything seems to be working but this is spammed in the logs

**Comment by TheDK** [comment]
Seeing this for the first time since updating to 2021.8.2:

```
2021-08-06 11:26:56 ERROR (MainThread) [frontend.js.latest.202108040] :0:0 ResizeObserver loop completed with undelivered notifications.
```

**Comment by smmalis37** [comment]
I'm seeing this too in 2021.9.6. It's a new install so no idea when it was introduced. Everything does seem to be working fine though, same as Final-Hawk above.

**Comment by fabian727** [comment]
I am seeing it too.
 "Home Assistant 2021.6.0" is natively installed on linux

Logger: frontend.js.latest.202106011
Source: components/system_log/__init__.py:190
:0:0 ResizeObserver loop completed with undelivered notifications.

**Comment by crixionz** [comment]
I get the same `ResizeObserver loop completed with undelivered notifications` every time I press `e` to make the entity search come up. Consistently.

**Comment by geokscott** [comment]
I've been seeing the same in my Logs for a few months now. It's triggered every time the Logbook is accessed.
```
Logger: frontend.js.latest.202106011
Source: components/system_log/init.py:190
:0:0 ResizeObserver loop completed with undelivered notifications.
```

**Comment by fabian727** [comment]
I did an update to 2021.11.3 now every ID is duplicated *_2 and working. The old one still exist and are unavailable.

**Comment by adekloet** [comment]
I have the errors opening Lovelace dashboards, it's annoying because my brain wants to go ahead and fix errors but I cannot pinpoint what to fix..
ERROR (MainThread) [frontend.js.latest.202111090] :0:0 ResizeObserver loop completed with undelivered notifications

**Comment by hwestman** [comment]
I keep getting these in the console when I trigger the [Quick Bar](https://www.home-assistant.io/docs/tools/quick-bar/).
It also seems to correlate with this error in the logs
```
Logger: frontend.js.latest.202111080
Source: components/system_log/__init__.py:190 
First occurred: 7:04:07 PM (17 occurrences) 
Last logged: 8:15:45 PM

:0:0 Script error.
```

Another observation is that the error only show up in my console on Safari and not Firefox. In Firefox the log message pops up in the HA log.
```
Logger: frontend.js.latest.202111080
Source: components/system_log/__init__.py:190
First occurred: 7:04:07 PM (19 occurrences)
Last logged: 8:24:53 PM

    :0:0 Script error.
    :0:0 ResizeObserver loop completed with undelivered notifications.

```

Running 2021.11.2

**Comment by mbo18** [comment]
Same here every time I click on Logbook in the sidebar:
`ERROR (MainThread) [frontend.js.latest.202112200] :0:0 ResizeObserver loop completed with undelivered notifications.`

**Comment by xconverge** [comment]
I think a frequent user facing error that can be ignored doesn't really make sense and it should be changed to a warning or be fixable

**Comment by mdegat01** [comment]
I've been seeing this for a bit too. I realize its ignorable since nothing has actually broken for me despite this (and plus bram says so) but it does add log noise. Plus you can't really filter it out since the logger is `frontend.js.latest.202203012`. Sure I can set that logger's level to `critical` but that's temporary until the next time frontend updates and the version updates. 

For reference I am using Firefox v98.0.2. I don't have any input numbers in my dashboards although I do have some input selects. I never really connected them to this issue but I have noticed some other frontend errors related to them so Marius's note on the input number made me think of them.

It would be nice if this was fixed. Or alternatively if it could at least use a consistent logger that doesn't change release to release so it can be filtered out.

**Comment by w35l3y** [comment]
In my case, it always happens when I type **slowly** in the search field of "Automation & Scenes"
If I type quickly, the error doesn't appear.

```
Logger: frontend.js.latest.202203012
Source: components/system_log/__init__.py:190
First occurred: 15:38:17 (2 occurrences)
Last logged: 15:46:54

:0:0 ResizeObserver loop completed with undelivered notifications.
```

![image](https://user-images.githubusercontent.com/963510/161396835-9da06381-5e11-4e2e-9f0d-d2b0a9365c72.png)
![image](https://user-images.githubusercontent.com/963510/161396798-5c134f58-d792-440f-8d1e-933a969e898b.png)


## System Health

version | core-2022.3.8
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.9.9
os_name | Linux
os_version | 5.10.103-v8
arch | aarch64
timezone | America/Recife

**Comment by fireheadman** [comment]
also have this issue....  just nice to eliminate errors while troubleshooting other things in HA

**Comment by davethepear** [comment]
still does it in 2022.6.7...

**Comment by TheDK** [comment]
Haven’t see this one in a while…

**Comment by xconverge** [comment]
I still see it constantly, and find it strange/cumbersome to have an "ERROR" (not a "WARNING") in my logs consistently and it is OK to ignore.

**Comment by steverep** [comment]
Testing in Firefox on Windows, I can no longer reproduce these errors after #15127 was merged.  This was an upgrade to our web component framework.

Just for some background edification, these errors are logged by the browser as mandated by the `ResizeObserver` specification.  This [section on MDN](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver#observation_errors) explains in more detail.

Resize observers are used by HA in:
- Virtualized data tables and scrollers, such as the entities or devices list, logbook, etc.  This is mostly out of our control as it is wrapped up in a dependency package.
- Independently as part of a handful of other components, such as number helpers, weather card, media card & controls, etc.  We'd have to go through these one instance at a time.



In the former virtualized areas, I believe this may be fixed.  Please test those areas in the upcoming 2023.2 release.

I'd say there's a lesser chance the latter areas are fixed, but I have not tested myself yet.

**Comment by steverep** [comment]
**Reminder:** If you're a beta tester, please look to see if this is resolved.

**Comment by Mariusthvdb** [comment]
hmm, I just managed to make it happen:

<img width="410" alt="Scherm­afbeelding 2023-01-27 om 09 35 41" src="https://user-images.githubusercontent.com/33354141/215043287-9f798eac-00e8-4de4-a0cf-6f708affec55.png">

but am clueless to the action it causes.
I had an inspector pane open in Safari Mac, on the Helpers page. Closed inspector, navigated to my Home page and reopened inspector, and there is was....

repeating that sequence doesnt reproduce.

**Comment by steverep** [comment]
So if it's occurring on a dashboard view, then it could be from any of the following:
- Weather card
- Calendar card
- Media control card
- Map card
- Entities card with either number helper or media player

All of those use a `ResizeObserver`.  I can take a look but if you can narrow down which one(s) to start with that would help.

**Comment by Mariusthvdb** [comment]
the one card I do not use is the calendar card.
of the others and related to what I tested earlier today, Id say the weather card and number helper would be a good start.

Having said that, I also noticed the error on views without any of these..

as a matter of fact, I just experienced it live! clicking an input_boolean to open the more-info on that. does that make any sense?

**Comment by steverep** [comment]
> the one card I do not use is the calendar card. of the others and related to what I tested earlier today, I'd say the weather card and number helper would be a good start.

Good to know.

> as a matter of fact, I just experienced it live! clicking an input_boolean to open the more-info on that. does that make any sense?

On a view, it would make sense if any of those cards were also on the page.  If you were on the helpers table though, it might mean the upgrades I mentioned earlier did not completely fix the virtualized areas.

**Comment by akula3** [comment]
Still persists on:
Home Assistant 2023.3.3
Supervisor 2023.03.1
Operating System 9.5
Frontend 20230309.0 - latest

**Comment by avinash6982** [comment]
I'm also facing the same issue, the error message is also blocking the view, not only on the console


![Screenshot 2023-03-29 at 11 57 00 AM](https://user-images.githubusercontent.com/92930097/228465793-d1f4e4a5-8852-4c0d-b5cb-d02489f5d8d7.png)

**Comment by Claudio1L** [comment]
![immagine](https://github.com/home-assistant/frontend/assets/97904190/57175259-f27d-4773-b2ab-bd72b3498014)


2023-05-13 16:19:46.046 ERROR (MainThread) [frontend.js.latest.202304111] :0:0 ResizeObserver loop completed with undelivered notifications.
2023-05-13 16:20:14.611 ERROR (MainThread) [frontend.js.latest.202304111] :0:0 ResizeObserver loop completed with undelivered notifications.
2023-05-13 16:20:35.841 ERROR (MainThread) [frontend.js.latest.202304111] :0:0 ResizeObserver loop completed with undelivered notifications.

**Comment by joseph-luketelo** [comment]
I am also getting the same error for my reactjs project. 
![Screenshot from 2023-05-23 09-33-14](https://github.com/home-assistant/frontend/assets/15190671/f25f67cf-3226-436f-9e55-f43d72fbd885)

**Comment by Mariusthvdb** [comment]
just noticed this isnt solved yet, even though I believe we were informed it was...

<img width="539" alt="Scherm­afbeelding 2023-08-06 om 23 01 05" src="https://github.com/home-assistant/frontend/assets/33354141/ceaf0ca2-d305-4227-a626-f0a4ab9bcbca">

popped up after clicking a sensor in an entities card in its purest form:

```
      - sensor.last_time_motion_triggered
```

<img width="588" alt="Scherm­afbeelding 2023-08-06 om 23 03 40" src="https://github.com/home-assistant/frontend/assets/33354141/d4c61baa-cf29-47a7-b58d-d994061f87b5">
